pkg/cloudprovider/providers/azure/azure_storage.go

```diff
@@ -24,51 +24,48 @@ import (
 
 // CreateFileShare creates a file share, using a matching storage account
 func (az *Cloud) CreateFileShare(name, storageAccount, storageType, location string, requestGB int) (string, string, error) {
-	var err error
+	var errResult error
 	accounts := []accountWithLocation{}
 	if len(storageAccount) > 0 {
 		accounts = append(accounts, accountWithLocation{Name: storageAccount})
 	} else {
 		// find a storage account
-		accounts, err = az.getStorageAccounts()
-		if err != nil {
+		accounts, errResult = az.getStorageAccounts()
+		if errResult != nil {
 			// TODO: create a storage account and container
-			return "", "", err
+			return "", "", errResult
 		}
 	}
 	for _, account := range accounts {
-		glog.V(4).Infof("account %s type %s location %s", account.Name, account.StorageType, account.Location)
+		glog.Errorf("account %s type %s location %s", account.Name, account.StorageType, account.Location)
```

Comment:
keep it as `glog.V(4).Infof`, it's not an error

Reply:
Good catch, added that for debugging, reverted.